src/components/TodoContainer.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+
+class TodoContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: uuidv4(),
+          title: 'Setup Development Environment 1',
+          completed: true,
+        },
+        {
+          id: uuidv4(),
+          title: 'Setup Development Environment 2',
+          completed: true,
+        },
+        {
+          id: uuidv4(),
+          title: 'Setup Development Environment 3',
+          completed: true,
+        },
+      ],
+    };
+  }
+
+  updateCompleted = (id) => {
+    this.setState((previousState) => (
+      {
+        todos: previousState.todos.map((todo) => {
```

Comment:
I don't remember very well, but I believe this can be done "___prettier___" by using the `useEffect` hook.

Here's the documentation if you're up to discussing if what I just said was true or not.

Reply:
We'll this class in particular has two event handlers which makes it a bit harder to refactor. Maybe it'll look prettier but somehow I feel I like this way better.

Although it's a good practice to refactor it